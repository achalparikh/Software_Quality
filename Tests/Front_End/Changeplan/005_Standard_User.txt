login
standard
John Doe
changeplan
John Doe
00012
logout