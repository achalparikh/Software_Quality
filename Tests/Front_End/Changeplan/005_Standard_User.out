Login to start:
Enter type of account: "standard" or "admin"
Logging in as standard user - enter name:
You are now logged in as "John Doe"
Enter a command:
Error, standard users cannot use the 'changeplan' transaction
Enter a command:
Error, invalid command, please enter a command:
Error, invalid command, please enter a command:
You are now logged out
Login to start:
